bin/run-tests

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+for D in exercises/*; do
+    CURRENT_DIR=$(pwd)
+
+    if [ -d "${D}" ]; then
+        # Get the exercise name from the test file
+        TEST_FILE=$(basename $(ls ${D}/test/test*))
```

Comment:
Okay, I think that this is really for getting the underscored name for the exercise (ie: `hello_world` instead of `hello-world`). I guess this raises the question of why we don't simply name the source exactly the way the exercise is named.